mappings/net/minecraft/client/util/math/Quaternion.mapping

```diff
@@ -1,4 +1,31 @@
 CLASS net/minecraft/class_1158 net/minecraft/client/util/math/Quaternion
 	FIELD field_5656 components [F
+	METHOD <init> (FFFF)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 w
+	METHOD <init> (FFFZ)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 w
+	METHOD <init> (Lnet/minecraft/class_1158;)V
+		ARG 1 other
+	METHOD <init> (Lnet/minecraft/class_1160;FZ)V
+		ARG 1 direction
+		ARG 2 magnitude
+		ARG 3 normalize
 	METHOD equals (Ljava/lang/Object;)Z
 		ARG 1 o
+	METHOD method_16002 fSin (F)F
+		ARG 0 value
+	METHOD method_16003 fCos (F)F
+		ARG 0 value
+	METHOD method_4921 x ()F
```

Comment:
```suggestion
	METHOD method_4921 getX ()F
```

Reply:
I don't want to commit these until I get an answer about the `x()`, `y()`, `z()` methods in all the vector classes.

Until then, I'd prefer to stick to the current convention.